tests/playwright/Report.md

```diff
@@ -0,0 +1,267 @@
+
+- Multi-browser and device (like iPhones with given screen sizes) support
+- jest-dev-server integration which can start your webserver like create-react-app before running the tests
+- expect-playwright integration which provides common expect helper functions
+
+jest-playwright is added to the jest configuration as a preset which makes all the features available to be used at your disposal. It was inspired by [jest-puppeteer](https://github.com/smooth-code/jest-puppeteer/blob/master/README.md#start-a-server).
+
+#### Example of jest-playwright.config.js
+
+```js
+module.exports = {
+ browsers: [process.env.BROWSER || "chromium"],
+ devices: ["iPhone 6", "Pixel 2"],
+};
+```
+
+Below are some great and useful functions that come with jest-playwright;
+
+Reset helper functions. These functions come in handy when you want to reset the browser or page for various reasons such as deleting cookies. It exposes functions like;
+
+- `resetPage()`
+- `resetContext()`
+- `resetBrowser()`
+
+Example usage of reset current browser
+
+```js
+beforeEach(async () => {
+ await jestPlaywright.resetBrowser();
+});
+```
+
+Debug helper functions. This help in debugging test incase there are test failure. It uses the `jestPlaywrightDebug()`
+
+Example usage of Debug
+
+```js
+test.jestPlaywrightDebug("failed", async ({ page }) => {
+ await page.goto("https://www.wikipedia.org/");
+ const title = await page.title();
+ await expect(title).toBe("Nothing");
+});
+```
+
+For more of these gems that come with jest-playwright, please visit the [Github Repo](https://github.com/playwright-community/jest-playwright)
+
+## Playwright Vs WebdriverIO
+
+### Statistics on WebdriverIO and Playwright Popularity
+
+![Screenshot](statistics.png)
+
+The above data was adopted from [npm trends](https://www.npmtrends.com/cypress-vs-playwright-vs-puppeteer-vs-selenium-vs-testcafe) showing some of the most popular automation test tools. From the above charts, we see that WebdriverIO still trumps Playwright when it comes to popularity among the testing community.
+
+### WebdriverIO and Playwright MediaWiki Core Tests Analysis
+
+![Screenshot](evaluation-chart.png)
+
+The above visualized chart represents data that was collected by running MediaWiki Core tests in WebdriverIO and Playwright. A total of 10 tests were run using both frameworks on [Github Actions](https://docs.github.com/en/actions) platform.
+
+The tests were run 40 times in both frameworks to ascertain stability and reliability. No flakiness and failures were encountered during the tests run. As shown above, there is consistency in time when the tests were run in WebdriverIO and Playwright.
+
+The above chart further shows that Playwright is much faster than WebdriverIO in terms of speed.
+
+### Why adapt Playwright
```

Comment:
Since wdio has sections Advantages and Disadvantages, it would make sense to add it here too.